content/en/docs/event-sources/kernel/_index.md

```diff
@@ -97,13 +54,16 @@ sudo bpftool feature probe kernel | grep -q "program_type tracing is available"
 
 ### How to run it
 
-The modern eBPF probe supports all the installation methods of other drivers:
+To enable the modern eBPF support in Falco, just set the `engine.kind` configuration key to `modern_ebpf`. Nothing else will be needed since no external artifact is required for it to work.
```

Comment:
Perhaps say something similar to https://github.com/falcosecurity/falco-website/pull/1246/files#diff-c3777e5fb1d24e6ce0aae8a99b12234f648c9ef9ed17e4b858fce246deffad5eR40